src/nvim/edit.c

```diff
@@ -8668,9 +8674,16 @@ static bool ins_tab(void)
     can_cindent = false;
   }
 
-  // When nothing special, insert TAB like a normal character
+  // When nothing special, insert TAB like a normal character.
+  bool ts_not_equal_sw = (tabstop_count(curbuf->b_p_vts_array) > 1)
```

Comment:
This was not in the patch.

Reply:
Yeah, I just got too annoyed formatting this fancy condition. Reverting.